.circleci/config.yml

```diff
@@ -4,297 +4,18 @@
-            ./cc-test-reporter before-build
-
-            bundle exec rake knapsack:rspec
-      - run:
-          name: Code Climate Test Coverage
-          command: |
-            aws s3 sync coverage/ "s3://login-gov-test-coverage/coverage-artifacts/$CIRCLE_BUILD_NUM/$CIRCLE_NODE_INDEX" --exclude '*' --include '.resultset.json' --include '.resultset.json.lock'
-            ./cc-test-reporter format-coverage -t simplecov -o "coverage/codeclimate.$CIRCLE_NODE_INDEX.json"
-            aws s3 sync coverage/ "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM"
-      # collect reports
-      - store_test_results:
-          path: /tmp/test-results
-      - store_artifacts:
-          path: /tmp/test-results
-          destination: test-results
-      - deploy:
-          command: |
-            aws s3 sync "s3://login-gov-test-coverage/coverage/$CIRCLE_BUILD_NUM" coverage/
-            ./cc-test-reporter sum-coverage --output - --parts $CIRCLE_NODE_TOTAL coverage/codeclimate.*.json | ./cc-test-reporter upload-coverage --input -
-            aws s3 sync "s3://login-gov-test-coverage/coverage-artifacts/$CIRCLE_BUILD_NUM" coverage/
-            bundle exec spec/simplecov_merger.rb
-            mkdir coverage_summary
-            mv coverage/index.html coverage_summary/
-            mv coverage/assets coverage_summary/
-            mv coverage/.resultset.json coverage_summary/
-            aws s3 --region us-west-2 rm s3://login-gov-test-coverage/coverage-artifacts/$CIRCLE_BUILD_NUM --recursive
-      - store_artifacts:
-          path: coverage_summary/
-
-  javascript_test:
-    working_directory: ~/identity-idp
-    executor: ruby_browsers
-    steps:
-      - install-browser-tools-no-firefox
-      - checkout
-      - node-install
-      - yarn-install
-      - run:
-          name: Run Tests
-          command: |
-            yarn test
-
-  javascript_build:
-    working_directory: ~/identity-idp
-    executor: ruby_browsers
-    environment:
-      NODE_ENV: production
-    steps:
-      - checkout
-      - node-install
-      - yarn-production-install
-      - bundle-install
-      - run:
-          name: Run Tests
-          command: |
-            bundle exec rake assets:precompile
-
-  lints:
-    working_directory: ~/identity-idp
-    executor: ruby_browsers
-    steps:
-      - checkout
-      - node-install
-      - yarn-install
-      - bundle-install
-      - run:
-          name: Run Lints
-          command: |
-            make lint
-  check-pinpoint-config:
-    executor: ruby_browsers
-    steps:
-      - checkout
-      - node-install
-      - yarn-install
-      - bundle-install
-      - run:
-          name: Check current AWS Pinpoint country support
-          command: |-
-            make lint_country_dialing_codes
-      - slack/status:
-          fail_only: true
-          failure_message: ':aws-emoji: :red_circle: AWS Pinpoint country configuration is out of date'
-  check_changelog:
+  placeholder:
```

Comment:
Just to clarify, is the intention to keep this around? I would have expected the config to be removed altogether.

Reply:
Nope.  The real step is to click the big red button on the CircleCI website, I just wanted this to be green for now since CircleCI complained when I deleted the file.